bin/node-template/node/src/chain_spec.rs

```diff
@@ -150,5 +150,6 @@ fn testnet_genesis(
 			// Assign network admin rights.
 			key: root_key,
 		},
+		transaction_payment: Default::default(),
```

Comment:
Was this somehow missed? I could not build https://github.com/paritytech/substrate/tree/monthly-2021-11 out of the box :grimacing:

Reply:
yes below in transaction payment we don't write the Config part, thus the genesis config of transaction payment is not included.
You can either do `exclude_part { Config }` or do as you did (which I think is better)

Reply:
yes default is good